packages/ckeditor5-image/tests/imageinsert/imageinsertui.js

```diff
@@ -278,6 +278,18 @@ describe( 'ImageInsertUI', () => {
 				sinon.assert.notCalled( commandSpy );
 				sinon.assert.calledOnce( cancelSpy );
 			} );
+
+			it( 'should focus on "insert image via URL" input after opening', () => {
+				let spy;
+
+				dropdown.on( 'change:isOpen', () => {
+					const first = dropdown.panelView.children.first;
+					spy = sinon.spy( first, 'focus' );
+				}, { priority: 'highest' } );
```

Comment:
```suggestion
                // The ImageInsertPanelView is added on first open.
                // See https://github.com/ckeditor/ckeditor5/pull/8019#discussion_r484069652
                dropdown.on( 'change:isOpen', () => {
                    const first = dropdown.panelView.children.first;
                    spy = sinon.spy( first, 'focus' );
                }, { priority: 'highest' } );
```

Reply:
Additionally, the `const first` might be either:

*   inlined
*   renamed to `imageInsertPanelView` to showcase which view it exactly checks.

That's because you would have to check the code to remember what's that first child.

Reply:
right, corrected